lib/NodeStatus.js

```diff
@@ -32,13 +33,18 @@ class NodeStatus {
   resetChangeGroups() {
     this.oldNode = null;
     // clone so that the next step's oldNode doesn't modify this step's newNode
-    this.newNode = clone(this.newNode, false);
-    this.newNode.filter(node => node.changeGroup).forEach(change => {
-      delete change.changeGroup;
-    });
+    this.newNode = NodeStatus.resetChangeGroups(this.newNode);
   }
 }
 
+NodeStatus.resetChangeGroups = function(node) {
```

Comment:
not sure if this is confusing that two `resetChangeGroups` have the same name (one can be called on a node status object, the other is a static function)